dinky-web/src/pages/RegCenter/Alert/AlertGroup/index.tsx

```diff
@@ -45,21 +47,18 @@ const AlertGroupTableList: React.FC = (props: any) => {
   /**
    * state
    */
-  const [modalVisible, handleModalVisible] = useState<boolean>(false);
-  const [updateModalVisible, handleUpdateModalVisible] = useState<boolean>(false);
-  const [formValues, setFormValues] = useState({});
-  const [alertGroupList, setAlertGroupList] = useState<Alert.AlertGroup[]>([]);
-  const [loading, setLoading] = useState<boolean>(false);
+  const [alertGroupState, setAlertGroupState] = useState<AlertGroupState>(InitAlertGroupState);
+
   const actionRef = useRef<ActionType>();
 
   /**
    * execute query alert instance list
    * set alert instance list
    */
   const queryAlertGroupList = async () => {
-    await queryList(API_CONSTANTS.ALERT_GROUP).then((res) => {
-      setAlertGroupList(res.data);
-    });
+    await queryList(API_CONSTANTS.ALERT_GROUP).then((res) =>
+      setAlertGroupState((prevState) => ({ ...prevState, alertGroupList: res.data }))
```

Comment:
For this, only using one of `await` or 'Promise` is better, recommend using `await`